aries_cloudagent/config/argparse.py

```diff
@@ -99,54 +99,73 @@ def add_arguments(self, parser: ArgumentParser):
             nargs=2,
             metavar=("<host>", "<port>"),
             env_var="ACAPY_ADMIN",
-            help="Specify the host and port on which to run the administrative server.\
-            If not provided, no admin server is made available.",
+            help=(
```

Comment:
DId you reformat all these descriptions or did Black do it?

Reply:
I did it. Black accepts them either way, but everywhere else in the code base they look like the style to which I've adapted them here.

Reply:
Let me just merge main again though.